sktime/classification/distance_based/_proximity_forest.py

```diff
@@ -155,9 +155,6 @@ def numba_wrapper(distance_measure):
     def distance(instance_a, instance_b, **params):
         instance_a = from_nested_to_2d_array(instance_a, return_numpy=True)
         instance_b = from_nested_to_2d_array(instance_b, return_numpy=True)
```

Comment:
would you mind using `convert(instance, from_type="nested_univ", to_type="numpyflat")` instead of `from_nested_to_2d_array`?

Also, are we sure you would like to pass a 2D array and not a 2D array here?

Reply:
so proximity forest is not part of the PR and wont be changed here, there is a PR for PF, coulkd you ask there instead?

Reply:
where is it?

Reply:
https://github.com/alan-turing-institute/sktime/pull/2112

Reply:
that is abandoned, per your statement

Reply:
its not abandoned, just delayed